examples/pytorch/speech-recognition/run_speech_recognition_ctc.py

```diff
@@ -0,0 +1,588 @@
+#!/usr/bin/env python3
+import functools
+import json
+import logging
+import os
+import re
+import sys
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional, Union
+
+import numpy as np
+import torch
+import torch.distributed as dist
+import torchaudio
+from datasets import DatasetDict, load_dataset, load_metric
+
+import transformers
+from transformers import (
+    AutoConfig,
+    AutoFeatureExtractor,
+    AutoModelForCTC,
+    AutoTokenizer,
+    HfArgumentParser,
+    Trainer,
+    TrainingArguments,
+    Wav2Vec2Processor,
+    set_seed,
+)
+from transformers.trainer_utils import get_last_checkpoint, is_main_process
+from transformers.utils import check_min_version
+from transformers.utils.versions import require_version
+
+
+# Will error if the minimal version of Transformers is not installed. Remove at your own risks.
+check_min_version("4.11.0.dev0")
+
+# TODO(Patrick) Bump up as soon as audio features are merged
+require_version("datasets>=1.12.0", "To fix: pip install -r examples/pytorch/text-classification/requirements.txt")
+
+
+""" Fine-tuning a 🤗 Transformers model for image classification"""
```

Comment:
```suggestion
""" Fine-tuning a 🤗 Transformers CTC model for automatic speech recognition"""
```

also, this should probably be at the top